rust/cardano-chain-follower/src/mithril_turbo_downloader.rs

```diff
@@ -245,6 +254,20 @@ impl Inner {
         }
         bail!("Can not deduplicate.");
     }
+
+    /// Open a file using mmap for performance.
+    fn mmap_open_sync(&self, path: &Path) -> MithrilResult<MemoryMapFile> {
+        match MemoryMapFile::try_from(path) {
+            Ok(mmap_file) => {
+                stats::update_mmap_count_and_size(self.cfg.chain, mmap_file.size());
```

Comment:
Do this inside MemoryMapFile,  so that anytime that type is used its stat is accumulated.